### PR TITLE
Opennic: css update to fix width issues

### PR DESCRIPTION
**Pull request by cylgom**
## Type of Change
- [ ] New Instant Answer
  - [ ] Cheat Sheet
- [X] Improvement
  - [X] Bug fix
  - [ ] Enhancement
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.)
## Description of new Instant Answer, or changes

This fixes the style issues with the Opennic IA (some site-wide style changes made it unreadable)
I already made some tests:
- Linux: Firefox
- OSX: Firefox, Safari
- Windows10: Firefox, Chrome, Opera, Edge, IE
- iOS: Dolphin, Safari

---

IA Page: https://duck.co/ia/view/open_nic

### CODE REVIEW

**Comment by daxtheduck** [comment]
@cylgom Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Sorry for the delay @cylgom -- deploying to Beta 👍

File: share/spice/open_nic/content.handlebars [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="opennic__list">
     <div class="opennic__subblocks opennic__ip"> {{ip}} </div>
     <div class="opennic__subblocks opennic__km"> {{#ifunit}} {{miles}} miles {{else}} {{kilometers}} km {{/ifunit}} </div>
-    <div class="opennic__subblocks opennic__stat"> {{stat}} % reliability </div>
+    <div class="opennic__subblocks opennic__stat"> {{stat}} % reliab. </div>
```
Comment (moollaza) [inline share/spice/open_nic/content.handlebars]: nit: the space before the `%` is a little funny..

**Pull request merged**